src/bun.js/bindings/webcore/AbortSignal.cpp

```diff
@@ -115,6 +115,15 @@ void AbortSignal::signalAbort(JSC::JSValue reason)
     dispatchEvent(Event::create(eventNames().abortEvent, Event::CanBubble::No, Event::IsCancelable::No));
 }
 
+void AbortSignal::cleanNativeBindings(void* ref) {
+    auto callbacks = std::exchange(m_native_callbacks, {});
+
+    callbacks.removeLastMatching([=](auto callback){
```

Comment:
should it be removeLastMatching or remove all matching?

Reply:
`removeAllMatching` is the right answer, fixed on https://github.com/oven-sh/bun/commit/483a9f85c384626b9c42740eb7a6421d79510367